demo/test_demos.py

```diff
@@ -17,6 +17,10 @@
 @pytest.mark.parametrize("scalar_type", ["float64", "float32", "complex128", "complex64"])
 def test_demo(file, scalar_type):
     """Test a demo."""
+    if sys.platform.startswith("win32") and "complex" in scalar_type:
+        # Skip complex demos on win32
+        pytest.skip(reason="_Complex not supported on Windows")
```

Comment:
What a pain . . . .

Reply:
Indeed, MSVC's lack of `_Complex` support has been the only major issue with doing a straightforward port.